bindings/matrix-sdk-ffi/src/room_alias.rs

```diff
@@ -0,0 +1,22 @@
+use regex::Regex;
+use ruma::RoomAliasId;
+
+/// Verifies the passed `String` matches the expected room alias format.
+#[matrix_sdk_ffi_macros::export]
+fn is_room_alias_format_valid(alias: String) -> bool {
+    RoomAliasId::parse(alias).is_ok()
+}
+
+/// Transforms a Room's display name into a valid room alias.
+#[matrix_sdk_ffi_macros::export]
+fn room_alias_from_room_display_name(room_name: String) -> String {
+    let whitespace_regex = Regex::new(r"\s+").unwrap();
```

Comment:
Let's put those strings into a const at the top of the function and let's then document what each of those do.

Additionally, let's replace the `unwrap()` with an `expect()`.